src/classes/Recipe.js

```diff
@@ -34,16 +34,21 @@ class Recipe {
   }
   
   listIngredients(ingredientData) {
-    return this.matchIngredients(ingredientData).map(element => `${+(element.quantity.amount)} ${element.quantity.unit} ${element.name}`);  
+    return this.matchIngredients(ingredientData).map(element => {
```

Comment:
Great fix on getting the quantity amount to format the way we wanted it to. Would love to find a way to simplify this - maybe we can dig back in for part 2?

Reply:
Great job on this solution!